### PR TITLE
Three things please

### LINKED ISSUE #808: Include built version in npm package
People with CSS build process will appreciate having the built version (the `dist/` directory) distributed with the npm package.

### LINKED ISSUE #838: Set <main> elements to display: block
IE11 does not support `<main>`: https://caniuse.com/#search=main

This does not frequently cause problems, or else we would have noticed before. But it *can* cause problems, because I just suffered from one of those problems this morning!

### LINKED ISSUE #839: scroll-styled should have momentum scrolling on iOS
To enable this, we need to add `-webkit-overflow-scrolling: touch;` to the rule.

### PR DESCRIPTION
**Pull request by davidtheclark**
- Momentum scrolling in iOS for scroll-styled …Closes #839.
- Reset main element to display block …Closes #838.
- Add dist to npm package …Closes #808.

@tristen for review.

After merging these I'd like to cut a new release.

### CODE REVIEW

File: src/miscellaneous.css [hunk 1]
```diff
@@ -14,6 +14,9 @@
  * <div class="scroll-styled scroll-auto h120"><div class='h240'></div></div>
  * <div class="scroll-styled scroll-styled--dark bg-gray-dark scroll-auto h120"><div class='h240'></div></div>
 */
+.scroll-styled {
```
Comment (tristen) [inline src/miscellaneous.css]: This doesn't strictly need to fall under `scroll-styled` right? Should this be added to `scroll-auto` instead?

**Review by tristen — COMMENTED** [review]
:one: 👏  :two: 👏 :three: 👏

File: src/miscellaneous.css (hunk 1 above)
Comment (davidtheclark) [reply src/miscellaneous.css]: I guess so, you're right.

**Comment by davidtheclark** [comment]
Tried some real Git-trickery to amend that first commit. I think I got it.

**Review by tristen — APPROVED** [review]

**Pull request merged**